### DISCUSSION TITLE
itemsjs

### DISCUSSION THREAD

**Discussion by tordans** [DefinitelyTyped/DefinitelyTyped#59705] [Request a new @types package]
https://github.com/itemsapi/itemsjs is a tool to add facetted search features to your frontend code. It would be great to have some shared types for that.

See also https://github.com/itemsapi/itemsjs/issues/106.

**Comment by MysteryBlokHed** [comment]
I'm available in a few hours to write types for this. I'll update with a PR number when it's done unless someone else writes types before then

**Reply by MysteryBlokHed** [reply]
Just opened #59717
